tensorbay/client/diff.py

```diff
@@ -121,12 +122,17 @@ class DataDiff(DiffBase):
 
     _T = TypeVar("_T", bound="DataDiff")
 
-    _repr_attrs = ("remote_path", "action")
+    _repr_attrs = ("action", "file", "label")
+
+    _repr_maxlevel = 2
 
     remote_path: str = attr(key=camel)
     file: FileDiff = attr()
     label: LabelDiff = attr()
 
+    def _repr_head(self) -> str:
+        return f'{self.__class__.__name__}("{self.remote_path}")'
```

Comment:
shall we rename `remote_path` to `path` due to RemoteData use `path` to represent `remote_path`